config/canonical.json

```
@@ -33483,8 +33483,11 @@
   },
   "tourism/hotel|Kyriad": {
     "count": 76,
+    "countryCodes": ["fr"],
     "tags": {
       "brand": "Kyriad",
+      "brand:wikidata": "Q529212",
+      "brand:wikipedia": "en:Groupe_du_Louvre",
```

I'd link to the French one here - also no `_` in the page title..
```suggestion
      "brand:wikipedia": "fr:Louvre Hotels Group",
```